.pre-commit-config.yaml

```diff
@@ -56,6 +61,11 @@ repos:
         args: ["--recursive", "--settings-path", "./pyproject.toml", "."]
         language: system
         types: [python]
+        exclude: |
+          (?x)^(
+              .*/gunicorn_conf.py|
+              .*/test_gunicorn_conf.py
+          )$
```

Comment:
Precommit and guincorn_conf were not playing nicely. 
MyPy and iSort kept failing-then succeeding, repeatedly.